client.go

```diff
@@ -626,3 +523,11 @@ func handleFallback(opts featureOption, featureName string, ctx *context.Context
 		Enabled: false,
 	}
 }
+
+func resolveToggle(snapshot *FeatureMemoryState, opts featureOption, featureName string) *api.Feature {
```

Comment:
This needs to be removed. This is a work around because the way we designed our storage interface means that end users implementing their own would lose access to caching. However, this is in the public API and it's hard and painful to remove now and this needs to be a different PR's problem